src/bios.rs

```diff
@@ -157,10 +159,62 @@ impl Component for Bios {
         crate::component::query_adopt_state()
     }
 
+    // Backup the current grub.cfg and install new static config
+    // - Backup "/boot/loader/grub.cfg" to "/boot/grub2/grub.cfg.bak"
+    // - Remove symlink "/boot/grub2/grub.cfg"
+    // - Install new static grub.cfg
+    // - Remove old "/boot/loader/grub.cfg"
+    fn migrate_static_grub_config(&self, sysroot_path: &str, destdir: &openat::Dir) -> Result<()> {
+        use anyhow::Context;
+        use openat_ext::OpenatDirExt;
+
+        let grub_config_dir = Utf8PathBuf::from(sysroot_path).join("boot/grub2");
+        let grubdir = destdir
```

Comment:
Hmm, I don't understand why we sometimes prefix `destdir` and sometimes not in the various path maniulations. If we want to assume that `destdir` can be something other than `/`, then I think some of the paths below need to be adjusted?

Reply:
Currently the destdir is `/`, and see https://github.com/coreos/bootupd/blob/main/src/bootupd.rs#L518 and [definition](https://github.com/coreos/bootupd/blob/main/src/bootupd.rs#L455).

Reply:
Agree it is confusing, update to "boot/grub2" to make it clear.